src/Frontend/Core/Header/Header.php

```diff
@@ -352,6 +352,20 @@ public function parse(): void
             $this->meta->addMetaData(MetaData::forName('robots', 'noindex, nofollow'), true);
         }
 
+        /**
+         * @remark only for SumoCoders
+         *
+         * Because ENV variables are not passed to CGI due to suExec.
+         * See http://wiki.dreamhost.com/Suexec#Apache_module_mod_env for more
+         * information.
+         */
+        if (isset($_SERVER['HTTP_HOST']) && substr_count($_SERVER['HTTP_HOST'], '.sumocoders.eu') >= 1) {
```

Comment:
I would do this in a different place

I would do this inside the MetaCollection in the constructor